gistic/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:14.04
+
+RUN apt-get update && apt-get upgrade --yes && \ 
+	apt-get install -y wget && \
+	apt-get install --yes bc vim libxpm4 libXext6 libXt6 libXmu6 libXp6 zip unzip  build-essential
+
+RUN mkdir /home/gistic
+WORKDIR /home/gistic
+
+RUN mkdir /home/gistic/MCRInstaller
+
+RUN cd /home/gistic/MCRInstaller && \
+   wget https://www.mathworks.com/supportfiles/downloads/R2014a/deployment_files/R2014a/installers/glnxa64/MCR_R2014a_glnxa64_installer.zip && \
+   unzip MCR_R2014a_glnxa64_installer.zip
+
```

Comment:
I'm not sure how, but also there are two copies of MCR installer zip, one here:
`/home/gistic/MCRInstaller` and here: `/home/gistic/MCR_Installer` the second one is weird.  I think both zipa should be removed after install, and that will shave another 1GB off the docker size.

Reply:
Thanks for checking ,will update